controller_manager/src/controller_manager.cpp

```diff
@@ -167,8 +167,8 @@ controller_interface::ControllerInterfaceSharedPtr ControllerManager::load_contr
   RCLCPP_INFO(get_logger(), "Loading controller '%s'", controller_name.c_str());
 
   if (!loader_->isClassAvailable(controller_type)) {
-    const std::string error_msg("Loader for controller '" + controller_name + "' not found");
-    RCLCPP_ERROR(get_logger(), "available classes:");
+    const std::string error_msg("Loader for controller '" + controller_name + "' Not found");
```

Comment:
Capitalize only begin of the message/sentence.

```suggestion
    const std::string error_msg("Loader for controller '" + controller_name + "' not found.");
```